<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chartwindow</class>
 <widget class="QWidget" name="chartwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="chart_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>711</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="performance_chart_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>711</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>0</y>
     <width>431</width>
     <height>581</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="Trades">
    <attribute name="title">
     <string>Trade List</string>
    </attribute>
    <widget class="QTableWidget" name="performanceTable">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>431</width>
       <height>541</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>68</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Entry</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Exit</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>%</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Result</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Drawup</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Drawdown</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Details</string>
    </attribute>
    <widget class="QTableWidget" name="detailsTable">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>530</width>
       <height>541</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::CurrentChanged</set>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>Wins</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Losses</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Win Rate</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Average Outcome</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Buy and Hold</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>General</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Long</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Short</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>320</y>
       <width>211</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string>A.O.: Win hoch: große Trades Lose hoch: möglicherweise macht ein Cut sinn.</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="minTextEdit">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>9</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="plainText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="maxTextEdit">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>9</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="plainText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="minus">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>16</y>
     <width>68</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxTextLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>16</y>
     <width>68</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>max:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="recalculateButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>9</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Recalculate</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Icy</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>9</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Strategy</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
